src/devices/portaudio/CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ yarp_prepare_plugin(portaudio
                     CATEGORY device
                     TYPE PortAudioDeviceDriver
                     INCLUDE PortAudioDeviceDriver.h
-                    EXTRA_CONFIG WRAPPER=grabber
+                    EXTRA_CONFIG WRAPPER=grabberDual
```

Comment:
This is not correct. Actually portaudio device has to be deprecated as well in favor of `portaudioPlayer` and `portaudioRecorder`. Functionalities of old portaudio has been split in two device, so there is no correct default wrapper now; it depends if you are using the old portaudio for recording or playing.
Anyway grabberDual for sure is wrong because it only supports images, not audio.

Reply:
Reverted this change